qmlti/src/lti/view/error.php

```diff
@@ -0,0 +1,23 @@
+<body class="bg-pattern" style="background-color: rgb(250, 250, 250);">
```

Comment:
This whole file needs to be adjusted to have single quotes everywhere

Reply:
IIRC HTML syntax is double quotes for primary items, such as classes. I've looked over the drupal generated HTML and it's also double quoted. PHP items are single quotes (I'll make that adjustment) but I think HTML consistency makes the most sense.